### PR TITLE
changed login to send username instead of email

### PR DESCRIPTION
**Pull request by FORDBA**
# Description
Please include a summary of the change and which issue is fixed / implemented. Please also include relevant motivation and context. List any dependencies that are required for this change (e.g., do you need to run an npm install to test?).
Fixes # (issue) # (issue2) ... (these should be the numbers of issues we have created / on the project board for this project)
## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# How Has This Been Tested?
Please describe step-by-step the process that a reviewer can follow to adequately test this PR. 
- [ ] Test to make sure Login is functional. This will require the matching server-side pull request to be checked simultaneously
- [ ] In dev tools, verify that login is sending both a token and the user ID
# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by mgjeffries — APPROVED** [review]
Looks Good!

File: src/components/auth/Login.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ export const Login = () => {
         Accept: "application/json",
       },
       body: JSON.stringify({
-        email: email.current.value,
+        username: username.current.value,
         password: password.current.value,
       }),
     })
```
Comment (mgjeffries) [inline src/components/auth/Login.js:25]: We should update the login process to set the userId and token (similar to how register currently works): 
localStorage.setItem("rare_user_token", res.token)
localStorage.setItem("rare_user_id", res.user_id)

**Review by Michaelr-Tyler — APPROVED** [review]

**Review by mgjeffries — CHANGES_REQUESTED** [review]

**Review by Michaelr-Tyler — COMMENTED** [review]
Just echoing what gib commented. Right now when you log in, it saves into local storage a   token as the rare_user_id.

We need the rare_user_id to equal the id
and then rare_user_token to equal the token of whoever logs in

**Review by mgjeffries — APPROVED** [review]
I verified that the user id is being set.

**Pull request merged**